Source/Core/Core/Core.cpp

```diff
@@ -98,12 +98,13 @@ static Common::Flag s_is_booting;
 static void* s_window_handle = nullptr;
 static std::string s_state_filename;
 static std::thread s_emu_thread;
-static StoppedCallbackFunc s_on_stopped_callback;
+Subscribable<State> g_on_state_changed;
```

Comment:
I already noticed that the initialization order for this can be wonky, would something like `Subscribable<State>& Core::OnStateChanged()` work for you?